src/System/Router/RouterLink.tsx

```diff
@@ -78,6 +76,12 @@ export const RouterAwareLink: React.FC<LinkPropsSimple & RouterLinkMixinProps> =
       return defaultValidatorFn(prop) || routerLinkValidator(prop)
     },
   })`
+    :hover {
+      color: ${props => props.inline && themeGet("colors.blue100")};
```

Comment:
@damassi `<Link>` wrapped around `<RouterLink>` internally still defaults to `RouterLink` styles. I ended up just conditionally adding the styles directly to RouterLink, but let me know if I misinterpreted what you were suggesting.